src/components/ChangelogDialog/i18n/en.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Changelog",
+  "link_docs": "Open changelog in documentation",
```

Comment:
Doesn't look like appropriate text for reusable component

Reply:
"View full changelog" would do

Reply:
Done